client/qiskit_serverless/core/job.py

```diff
@@ -572,7 +572,7 @@ def filtered_logs(self, **kwargs) -> str:
         """
         return self._job_client.filtered_logs(job_id=self.job_id, **kwargs)
 
-    def result(self, wait=True, cadence=5, verbose=False):
+    def result(self, wait=True, cadence=5, verbose=False, maxwait=0):
```

Comment:
has it sense to just set `wait=False` by default and that's all? I think the final output is similar and it has more sense for me.

Reply:
The scan suggested that loops needs static termination condition.  So I added not to wait forever.